.murdock

```diff
@@ -469,12 +470,17 @@ compile() {
         make -C${appdir} clean all test-input-hash -j${JOBS:-4}
```

Comment:
maybe the "-j4" here causes an issue? the "test-input-hash" target doesn't depend on `${CLEAN}`, so it might pick up an old hashed file before clean cleans it and all causes a rebuild?
(to fix, split "clean", or "test-input-hash", in a seperate line?